WalletWasabi.Fluent/ViewModels/Wallets/Actions/Receive/ReceiveWalletActionViewModel.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using System.Windows.Input;
+using ReactiveUI;
+using WalletWasabi.Blockchain.Analysis.Clustering;
+using WalletWasabi.Blockchain.Keys;
+using WalletWasabi.Fluent.ViewModels.NavBar;
+using WalletWasabi.Stores;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.ViewModels.Wallets.Actions.Receive
+{
+	[NavigationMetaData(
+		Title = "Receive",
+		Caption = "",
+		IconName = "wallet_action_receive",
+		NavBarPosition = NavBarPosition.None,
+		Searchable = false,
+		NavigationTarget = NavigationTarget.DialogScreen)]
+	public partial class ReceiveWalletActionViewModel : WalletActionViewModel
```

Comment:
Please rename this class to ReceiveViewModel (base class does nothing, should be able to remove that also.

Reply:
`WalletActionViewModel` has the `IsEnable` property which is important because users should not be able to use Receive while the wallet is not loaded (if loaded `IsEnable` turns to true)